scripts/generate_index_pages.ts

```diff
@@ -151,37 +144,31 @@ async function main() {
     );
   }
 
-  for (const stateCode in STATES) {
-    const stateName = (STATES as any)[stateCode];
-    const relativeSiteUrl = `/us/${stateCode.toLowerCase()}/`;
-    const relativeImageUrl = `/states/${stateCode.toLowerCase()}`;
-    await buildLocationPages(
-      builder,
-      relativeSiteUrl,
-      relativeImageUrl,
-      stateName,
-    );
-  }
-
-  for (const fips of COUNTIES) {
-    if (BLACKLISTED_COUNTIES.includes(fips)) {
-      continue;
-    }
-    const county = findCountyByFips(fips);
-    assert(county, 'Failed to find county ' + fips);
-    const stateCode = county.state_code;
-    const locationName = `${county.county}, ${(STATES as any)[stateCode]}`;
-    const relativeSiteUrl = `/us/${stateCode.toLowerCase()}/county/${
-      county.county_url_name
-    }`;
-    const relativeImageUrl = `counties/${fips}`;
+  regions.states.forEach(async (state: State) => {
```

Comment:
Hrm.  I think forEach() isn't promise-aware, which means this is going to end up doing all states in parallel, not wait for them to complete (but of course node won't end the process until they do), and silently ignore any promise failures.

So I'd just do a normal `for (const state of regions.states) {` loop...  Or if you really do want to do them in parallel, I guess:

```
await Promise.all(regions.states.map(async state => { 
 ...
}));
```

Reply:
Oh that's good to know, I'm changing it back to use the `for` loop!